procrustes/psdp.py

```diff
@@ -33,10 +33,138 @@
 
 __all__ = [
     "psdp_woodgate",
-    "psdp_peng"
+    "psdp_peng",
+    "psdp_opt"
```

Comment:
We should put it as 
```python
psdp_opt,
```
By doing so, you can keep this line untouched if you add new functions in the future.